app/src/androidTest/java/org/kiwix/kiwixmobile/common/data/database/KiwixDatabaseTest.java

```diff
@@ -17,7 +17,7 @@
  * MA 02110-1301, USA.
  */
 
-package org.kiwix.kiwixmobile.database;
+package org.kiwix.kiwixmobile.common.data.database;
```

Comment:
Why is `common` package been added? I have also been refactoring the code a bit as we discussed.

Reply:
the "common" package contains everything that is used by the whole appication. for example the database and the network module for api calls. these all can be used from anywhere inside the app and so are not module specific.